TestingExamples/Common/TestProjects.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Common;
+
+public static class TestProjects
+{
+    public static readonly string ApiTesting = "ApiTesting";
```

Comment:
This has nothing to do with your code here, but I've got a generic base UI test together using the extensions package, so once this PR is wrapped up I'll probably bring those changes in and add a UiTesting project type here too

Reply:
That sounds awesome